lte/gateway/c/session_manager/PipelinedClient.cpp

```diff
@@ -319,11 +303,36 @@ bool AsyncPipelinedClient::add_gy_final_action_flow(
       });
   return true;
 }
+
+void AsyncPipelinedClient::setup_policy_rpc(
+    const SetupPolicyRequest& request,
+    std::function<void(Status, SetupFlowsResult)> callback) {
+  auto local_resp = new AsyncLocalResponse<SetupFlowsResult>(
+      std::move(callback), RESPONSE_TIMEOUT);
+  PrintGrpcMessage(
```

Comment:
nit: not related to this change but can we update the naming of this function? `printGrpcMessage` to follow camelcase naming of C++ functions, maybe this can be added as a TODO

Reply:
Thanks, I will change that on a different PR